scipy/spatial/transform/_rotation.pyx

```diff
@@ -1273,6 +1386,185 @@ cdef class Rotation:
         else:
             return cls(quat, normalize=False, copy=False)
 
+    @cython.embedsignature(True)
+    @classmethod
+    def from_davenport(cls, axes, angles, extrinsic, degrees=False):
```

Comment:
@nmayorov do you think I should switch the order here from `axes, angles, extrinsic` to `axes, extrinsic, angles`?

Reply:
I think it will be more in line with `from_euler`. Sounds like a good idea.

Another option is to remove it and sort of rely on straightforward "apply in order" logic, but then if it appears in `as_davenport` it might be confusing.

Reply:
> Another option is to remove it and sort of rely on straightforward "apply in order" logic, but then if it appears in as_davenport it might be confusing.
The problem is that this wouldn't be symmetric with how `as_euler` and `from_euler` are implemented, since they also consider extrinsic and intrinsic options.

If I change the order, I think it could also be confusing since I'd have to call something like:
```python
Rotation.from_davenport([0,0,1], True, 90, degrees=True)
```
Since stating `extrinsic=True` would throw a Python error `SyntaxError: positional argument follows keyword argument`, but not stating `extrinsic` might be confusing.

Equivalently, instead of a `bool`, I could accept a `str`, maybe called `order`, that accepts either `'extrinsic'` or `"intrinsic"`, and then:
```python
Rotation.from_davenport([0,0,1], 'extrinsic', 90, degrees=True)
```

Reply:
And maybe also `'i'` and `'e'`

Reply:
String will be more intuitive, I think it's fine to accept both "extrinsic"/"intrinsic" and "e"/"i".

Reply:
Done